### DISCUSSION TITLE
Localized submit_date in JavaScript plugin

### DISCUSSION THREAD

**Discussion by Loneattic** [danirus/django-comments-xtd#246] [Q&A]
how can I localize the submit_date when using the JavaScript plugin?

**Comment by danirus** [comment]
You get the `submit_date` after the frontend request the list of comments. The list of comments is requested from the `commentbox.js` module (in the `static/django_comments_xtd/js/src` directory), in the function `load_comments`. The `load_comments` calls the `create_tree` that creates the `tree` state variable. Later in the `render` function creates a `<Comment />` component out of each item in the `tree`. All this happens in the `commentbox.js` module. The `<Comment />` component is in the separate module `comment.js`, in the same directory as `commentbox.js`.

I'm not sure whether that's what you wanted. Let me know otherwise.

**Reply by Loneattic** [reply]
Thanks for the answer.
Plugin gets the UTC date, as specified in the Django settings. Usually the date in the template adjusts itself for each user's local time. But comments display it in UTC.

**Reply by danirus** [reply]
Good catch. You sent this as an issue and I converted it into a conversation. I can not take that action back. Would you be so kind as to create a new issue for this? Thank you!
